scripts/items/evokers_gages.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+-- ID: 14960
+-- Evoker's Gages
+-- Pet mod via latent effect
+-----------------------------------
+local itemObject = {}
+local listenerPrefix = 'PET_MOD_LATENT'
+local latentPetId = xi.petId.LEVIATHAN
+local latentMods =
+{
+    { xi.mod.ATT, 10 },
+}
+
+itemObject.onItemCheck = function(target)
+    return 0
+end
+
+itemObject.onItemEquip  = function(user, item)
+    local listenerName = fmt('{}_{}', listenerPrefix, item:getID())
+    xi.itemUtils.handlePetLatentMods(user, latentPetId, latentMods, true)
+
+    user:addListener('MAGIC_STATE_EXIT', listenerName, function(player, spell)
+        if
+            spell:getSpellGroup() == xi.magic.spellGroup.SUMMONING
+        then
```

Comment:
There is no need to divide into blocks with only one condition.
`if spell:getSpellGroup() == xi.magic.spellGroup.SUMMONING then`

This applies to all items here and in your previous PR, since this is a continuation

Reply:
:(